tests/routes/test_validation_api.py

```
@@ -1,4 +1,7 @@
 from nanoid import generate as generate_nanoid
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)